### PR TITLE
added resx to education/create validator

### PR DESCRIPTION
**Pull request by a5x55555**

### CODE REVIEW

File: src/EducationService.Validation/Education/Resources/EducationValidatorResource.resx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+		<value>1.3</value>
+	</resheader>
+	<resheader name="reader">
+		<value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=2.0.3500.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+	</resheader>
+	<resheader name="writer">
+		<value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=2.0.3500.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+	</resheader>
+	<data name="IsEmpty" xml:space="preserve">
+		<value>Can't be empty</value>
```
Comment (vorobevamv) [inline src/EducationService.Validation/Education/Resources/EducationValidatorResource.resx]: as you use it with field name, start with lowercase letter

File: src/EducationService.Validation/Education/CreateEducationRequestValidator.cs [hunk 2]
```diff
@@ -23,21 +26,23 @@ public class CreateEducationRequestValidator : AbstractValidator<CreateEducation
...
+      
+      Thread.CurrentThread.CurrentUICulture = new CultureInfo("ru-RU");
 
       RuleFor(x => x.UserId)
         .Cascade(CascadeMode.Stop)
-        .NotEmpty().WithMessage("Wrong user id value.")
+        .NotEmpty().WithMessage($"{nameof(CreateEducationRequest.UserId)} {EducationValidatorResource.IsEmpty}")
         .MustAsync(async (pu, cancellation) => await CheckValidityUserId(pu, new List<string>()))
-        .WithMessage("User does not exist.");
+        .WithMessage($"{nameof(CreateEducationRequest.UserId)} {EducationValidatorResource.UserDoesNotExist}");
```
Comment (vorobevamv) [inline src/EducationService.Validation/Education/CreateEducationRequestValidator.cs]: may be without {nameof(CreateEducationRequest.UserId)}?
what would be the full message?

File: src/EducationService.Validation/Education/Resources/EducationValidatorResource.resx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+	</data>
+	<data name="UserDoesNotExist" xml:space="preserve">
+		<value>This user doesn't exist</value>
+	</data>
+	<data name="IsNotInEnum" xml:space="preserve">
+		<value>This value isn't a valid enum value</value>
```
Comment (vorobevamv) [inline src/EducationService.Validation/Education/Resources/EducationValidatorResource.resx]: User does'n know what is "enum"

File: src/EducationService.Validation/Education/Resources/EducationValidatorResource.ru-RU.resx [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+        <value>1.3</value>
+    </resheader>
+    <resheader name="reader">
+        <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=2.0.3500.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+    </resheader>
+    <resheader name="writer">
+        <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=2.0.3500.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+    </resheader>
+    <data name="IsEmpty" xml:space="preserve">
+        <value>Не может быть пустым</value>
```
Comment (vorobevamv) [inline src/EducationService.Validation/Education/Resources/EducationValidatorResource.ru-RU.resx]: start with lowercase letter

File: src/EducationService.Validation/Education/CreateEducationRequestValidator.cs [hunk 5]
```diff
@@ -23,21 +26,23 @@ public class CreateEducationRequestValidator : AbstractValidator<CreateEducation
...
+      
+      Thread.CurrentThread.CurrentUICulture = new CultureInfo("ru-RU");
 
       RuleFor(x => x.UserId)
         .Cascade(CascadeMode.Stop)
-        .NotEmpty().WithMessage("Wrong user id value.")
+        .NotEmpty().WithMessage($"{nameof(CreateEducationRequest.UserId)} {EducationValidatorResource.IsEmpty}")
         .MustAsync(async (pu, cancellation) => await CheckValidityUserId(pu, new List<string>()))
-        .WithMessage("User does not exist.");
+        .WithMessage($"{EducationValidatorResource.UserDoesNotExist}");
```
Comment (vorobevamv) [inline src/EducationService.Validation/Education/CreateEducationRequestValidator.cs]: .WithMessage(EducationValidatorResource.UserDoesNotExist);

**Review by vorobevamv — APPROVED** [review]

**Review by VLiamin — APPROVED** [review]

**Pull request merged**